go-getting-started/config/connect.go

```diff
@@ -0,0 +1,26 @@
+package config
```

Comment:
Package `config` is usually used for reading environment variables. 
Your `ConnectDB` function is responsible for connecting to Mongo, hence should be located in a `/storage/mongo` package.